### PR TITLE
Add hint to beekeeper no hive message

### PR DESCRIPTION
**Pull request by dhouck**
It's not obvious what to do when there's no hive for the beekeeper to work at, so explain that the hive tool is needed.

Closes #
Closes #
Closes #

# Changes proposed in this pull request:
-
-
-

Review please

### CODE REVIEW

**Review by marchermans — DISMISSED** [review]

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -1749,7 +1749,7 @@
   "com.minecolonies.coremod.gui.beekeeper.collectcomb": "Collect honeycomb",
   "com.minecolonies.coremod.gui.beekeeper.collectboth": "Collect comb and honey",
   "com.minecolonies.coremod.gui.beekeeper.addhives": "Get hive tool",
-  "entity.beekeeper.messagenohives": "There are no hives for me to work at!",
+  "entity.beekeeper.messagenohives": "There are no hives for me to work at!  Please assign me some using the hive tool available from my hut.",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "entity.beekeeper.messagenohives": "Either there are no hives for me to work at, or you haven't assigned me any. Use the hive tool available from my hut to do so.",
```

**Review by ravenbuilder934 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/minecolonies/lang/en_us.json (hunk 1 above)
Comment (Raycoms) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: Use the hive tool available from my hut to do so. ? else it sounds repetitive

Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: Good point

**Comment by dhouck** [comment]
Yeah, the suggested wording changes were good; updated

**Review by ravenbuilder934 — APPROVED** [review]

**Review by Mekle001 — APPROVED** [review]

**Review by marchermans — APPROVED** [review]

**Pull request merged**